app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PostsController < ApplicationController
+  # Create action - Handle the creation of a new post
+
+  def create
+    @user = User.find(params[:author_id])
+    @post = @user.posts.create(post_params)
+    redirect_to user_path(@user)
+  end
+
```

Comment:
- [ ] You don't have any create route for posts, so this action is somewhat useless. And it is not part of this project. Kindly remove this controller action.